src/instrumentsscene/qml/MuseScore/InstrumentsScene/ChooseInstrumentsPage.qml

```diff
@@ -142,6 +142,10 @@ Rectangle {
 
             onAddSelectedInstrumentsToScoreRequested: {
                 prv.addSelectedInstrumentsToScore()
+                if(!root.canSelectMultipleInstruments && invoker)
+                {
+                    invoker.submit()
+                }
```

Comment:
And here, you would "emit" that signal:
```qml
    root.submitRequested()
```
(and don't forget to remove the check for `invoker` too)